### PR TITLE
Added test for go formating

### PR DESCRIPTION
**Pull request by hibooboo2**

### CODE REVIEW

File: scripts/test [hunk 1]
```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
 
+result=$(find . -regex ".*go" | grep -iv godeps | xargs gofmt -s -l)
```
Comment (ibuildthecloud) [inline scripts/test]: Change to `find -name "*.go"`

File: scripts/test [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+for i in $result; do
+    echo $i
+done
+[ ! -z "$result" ] && exit 1
```
Comment (ibuildthecloud) [inline scripts/test]: Change to `[ -n "..." ]`

File: scripts/test [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+
 echo OK
+exit 0
```
Comment (ibuildthecloud) [inline scripts/test]: Not needed

File: scripts/test [hunk 4]
```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
```
Comment (ibuildthecloud) [inline scripts/test]: In general add `set -e`

File: scripts/test (hunk 1 above)
Comment (ibuildthecloud) [inline scripts/test]: `grep -v Godeps`

**Comment by hibooboo2** [comment]
Fixed

**Comment by ibuildthecloud** [comment]
Looks good, squash the commits

**Comment by ibuildthecloud** [comment]
@hibooboo2 squash and I'll merge this

**Comment by hibooboo2** [comment]
@ibuildthecloud Squashed it

**Pull request merged**